Dockerfile.prow

```diff
@@ -4,6 +4,19 @@ FROM registry.ci.openshift.org/open-cluster-management/builder:go1.16-linux AS b
 
 WORKDIR /workspace
 COPY . .
+
+# install yarn to build binary
+RUN nodejs_version=v14.16.1
```

Comment:
Do not do that, because it will make the thing complex in midstream build. Please refer to openshift/prometheus. I guess they should build in local and upload the binary in repo.